### DISCUSSION TITLE
How to make a next line in the node label?

### DISCUSSION THREAD

**Discussion by AliceSum** [xyflow/xyflow#4655] [Q&A]
For example, you can see Node 1 in this example.
However, I want to do it like 
Node
1

Yes. The 1 is in the second line and the Node is in the first line. I tried n\, <be>, and more. None of them works.

**Comment by santosh-sohani** [comment]
- Just create a custom component
- Use the `data` object with label something like this

```js
   data :  {
                 label1: 'Node',
                 label2:'1'
           }
```

This will be helpful
